runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2953,6 +2957,11 @@ gcInitializeDefaults(J9JavaVM* vm)
 	/* omrVM->gcPolicy is set by configurateGCWithPolicyAndOptions */
 	((J9JavaVM*)env.getLanguageVM())->gcPolicy = vm->omrVM->gcPolicy;
 
+	initializeIndexableObjectHeaderSizes(vm);
+	extensions->indexableObjectModel.setHeaderSizes(vm);
+#if defined(J9VM_ENV_DATA64)
+	extensions->indexableObjectModel.setIsIndexableDataAddrPresent(vm);
+#endif /* J9VM_ENV_DATA64 */
```

Comment:
Please be consistent with the closing comments; this should be like line 3078:
```c
#endif /* defined(J9VM_ENV_DATA64) */
```

Reply:
Ok. I was trying to be consistent with the majority of the #endif comments in this file, but I will make it like 3078.